OddNumberSum.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+
+int main()
+
+{
+   int list[] = { 7, 4, 1, 5, 3, 2, 7, 6, 4, 2, 9, 1, 2, 6, 8, 2, 3, 4 };
+
+   int array = sizeof(list);
```

Comment:
This is called `array`, but it holds a size. I think the name needs to be improved.